File: proyecto_java/ProblematicaJ/src/funciontenis/funciontenis.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package funciontenis;

import java.util.Scanner;

/**
 *
 * @author carlo
 */
public class funciontenis {

    public static String registrarParticipanteTenis() {
        Scanner entrada = new Scanner(System.in);
        String nombre;
        int edad;
        String nivel;
        String ciudad;
        String mano;
        String reporte = "";

        System.out.println("Ingrese nombre del participante");
        nombre = entrada.nextLine();
        System.out.println("Ingrese la edad del participante");
        edad = entrada.nextInt();
        entrada.nextLine();
        System.out.println("Ingrese el nivel: (Principiante, Intermedio, "
                + "Avanzado)");
        nivel = entrada.nextLine();
        System.out.println("Ingrese la ciudad del participante");
        ciudad = entrada.nextLine();
        System.out.println("Ingrese la mano hábil: (Derecha, Izquierda)");
        mano = entrada.nextLine();
        reporte = String.format("%s\n" + "Participante: %s\n"
                + "Edad: %d\n"
                + "Nivel: %s\n"
                + "Ciudad: %s\n"
                + "Mano hábil: %s\n", reporte, nombre, edad, nivel, ciudad, mano);
        return reporte;
    }
}

File: proyecto_python/problematica/funcionTenis.py
def registrarParticipanteTenis():
    nombre = (input("Ingrese nombre del participante: "))
    edad = (int(input("Ingrese edad del participante: ")))
    nivel = (input("Ingrese nivel del participante: (Principiante, Intermedio, Avanzado): "))
    ciudad = (input("Ingrese ciudad del participante: "))
    mano = (input("Ingrese mano habil del participante:(Derecha/Izquierda): "))

    reporte = (
    "Participante: " + nombre 
    + "\nEdad: " + str(edad) 
    + "\nPosición preferida: " + nivel 
    + "\nCiudad: " + ciudad 
    + "\nEstilo favorito: " + mano + "\n\n")

    return reporte